### ISSUE TITLE
has(::TargetIterator, name::String) deprecation warning

### ISSUE THREAD

**Issue by CarloLucibello** [JuliaGPU/CUDA.jl#271]
On Flux master, after [moving to CUDA.jl](https://github.com/FluxML/Flux.jl/pull/1258), I get the following warning when `using Flux`:
```julia 
using Flux
┌ Warning: key`has(::TargetIterator, name::String)` is deprecated, use `Target(; name = name) !== nothing` instead.
│   caller = llvm_compat(::VersionNumber) at compatibility.jl:181
└ @ CUDA ~/.julia/packages/CUDA/42B9G/deps/compatibility.jl:181
```
This doesn't happen when only `using CUDA`, but I can't trace the origin of this warning in Flux's code.

**Comment by maleadt** [comment]
Depwarns aren't issues... Besides, this is "fixed" on master already. The warning will be gone in the next version.

**Comment by maleadt** [comment]
@CarloLucibello This should be now be available in a tagged release, v1.1.
